use-cases/rag-on-gke/embedding-models/text-embedding/src/requirements.txt

```diff
@@ -0,0 +1,6 @@
+sentence_transformers>=2.7.0
+Flask
+requests
+waitress
```

Comment:
Can you alphabetize these, pin them to a version, and remove the additional empty lines?